### ISSUE TITLE
Unblocking pull requests

### ISSUE THREAD

**Issue by npmccallum** [sfackler/rust-openssl#1245]
There are a number of pull requests for this (very important) crate that are going unanswered. For example:

https://github.com/sfackler/rust-openssl/pull/1234
https://github.com/sfackler/rust-openssl/pull/1095
https://github.com/sfackler/rust-openssl/pull/1123
https://github.com/sfackler/rust-openssl/pull/1214
https://github.com/sfackler/rust-openssl/pull/1194
https://github.com/sfackler/rust-openssl/pull/1044
https://github.com/sfackler/rust-openssl/pull/1074

@sfackler I understand that you might not have time to work on this anymore. Would you be open to accepting other contributors with merge access? For example, the following people had some involvement in the above patches. Perhaps some of them would be willing to put in some review time to help clear the backlog?

@ueno
@kubo39 
@landaire
@dangerousplay
@Firstyear
@jrobsonchase
@pedrotbtl
@mbr
@fengyc
@mingjunyang
@pyfqm
@fleimgruber
@abcfy2
@laumann
@benmmurphy
@Leo1003
@mbelop

**Comment by Firstyear** [comment]
I'm happy to assist here and increase my involvement in the project @sfackler :)

**Comment by Leo1003** [comment]
I think I am familiar enough with OpenSSL. I am also willing to attend. :rocket:

**Comment by laumann** [comment]
@npmccallum Thanks for bringing more attention to this - I just checked back, because I would really like to have #896, so I'll try to rebase that and open another PR.
